gradle.properties

```diff
@@ -8,4 +8,4 @@ name = logisim-evolution
 #  x.y.z-suffix (i.e. "3.6.9-beta1")
 # Suffix is optional and can be any string indicating **non-stable** releases
 # (i.e. "beta1" or "rc1"). Keep suffix EMPTY string for every stable release!
-version = 3.6.0
```

Comment:
Not sure `develop` suffix is a best choice. I'd rather use `rc1` here, and the rationale is that `develop` means nothing really while `rc` is commonly used. Also `develop` is the branch name and nightly builder adds branch name to artifacts file names so double `develop` may look odd.

Reply:
So I just make 3.7.0 without extention.

Reply:
I think that's fine. As long as we release from `rc_3_6_0` (or from `master` with merged `rc_3_6_0`) we should be good. Version in `develop` is fine.